ports/mimxrt10xx/background.c

```diff
@@ -28,10 +28,14 @@
 #include "supervisor/port.h"
 
 void port_background_task(void) {
+}
+
+void port_background_tick(void) {
```

Comment:
Is this going to run a lot more often because it's on tick rather than task? Is that OK?